seekinternational/notification-library
# Feature/facebook feed

What does this PR do?
---------------------
Add Facebook feed function.

How can I test this?
--------------------
1) Configure `accessToken` in `fbConfigure` to use page token.
2) Call the feed function by passing: 

```
{
  message: 'message'
}
```
3) Check the FB page.

![image](https://user-images.githubusercontent.com/8541235/33921650-3151eff0-e000-11e7-8ae5-b8b7781272fe.png)
